sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2461,26 +2461,34 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
         s"but it's ${endSeconds.toString} now.")
   }
 
-  def failedToReadDeltaFileError(fileToRead: Path, clazz: String, keySize: Int): Throwable = {
+  def failedToReadDeltaFileError(fileToRead: Path, clazz: String, message: String): Throwable = {
     new SparkException(
-      errorClass = "_LEGACY_ERROR_TEMP_2258",
+      errorClass = "CANNOT_LOAD_STATE_STORE.CANNOT_READ_DELTA_FILE",
       messageParameters = Map(
         "fileToRead" -> fileToRead.toString(),
         "clazz" -> clazz,
-        "keySize" -> keySize.toString()),
+        "message" -> message),
       cause = null)
   }
 
   def failedToReadSnapshotFileError(fileToRead: Path, clazz: String, message: String): Throwable = {
     new SparkException(
-      errorClass = "_LEGACY_ERROR_TEMP_2259",
+      errorClass = "CANNOT_LOAD_STATE_STORE.CANNOT_READ_SNAPSHOT_FILE",
       messageParameters = Map(
         "fileToRead" -> fileToRead.toString(),
         "clazz" -> clazz,
         "message" -> message),
```

Comment:
Please, avoid arbitrary text in message parameters. As I can see, `message` can be:
1.
```scala
          throw QueryExecutionErrors.failedToReadSnapshotFileError(
            fileToRead, toString(), s"key size cannot be $keySize")
```

2.
```scala
            throw QueryExecutionErrors.failedToReadSnapshotFileError(
              fileToRead, toString(), s"value size cannot be $valueSize")
```

Please, put error message text to error-classes.json